identity/conf/routes

```diff
@@ -71,7 +71,7 @@ GET         /adverts/manage                         controllers.AdvertsManager.r
 # Consents journey
 ########################################################################################################################
 GET         /consents/thank-you                     controllers.editprofile.EditProfileController.displayConsentsJourneyThankYou
-GET         /consents/staywithus                    controllers.editprofile.EditProfileController.displayConsentsJourneyGdprCampaign
+GET         /consents/staywithus                    controllers.editprofile.EditProfileController.displayConsentsJourney(consentHint: Option[String])
```

Comment:
could this be a redirect instead? (like `displayPrivacyFormRedirect`) Even if we do update the branding this URL no longer makes sense